sdk/storage/Azure.Storage.Common/src/Shared/ClientsideEncryption/Models/EncryptionDataSerializer.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+
+namespace Azure.Storage.Cryptography.Models
```

Comment:
It'd be nice to have some comments on some of these methods to visualize what's happening.

Reply:
Added in upcoming push.